cactusbot/cactus.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, service):
 
         self.service = service
 
-    async def run(self, api, *auth):
```

Comment:
This should stay as `*auth` - remember, service agnostic. (Discord needs two parameters, for example.)